eng/pipelines/notifications.yml

```diff
@@ -0,0 +1,82 @@
+# Configure notifications in Azure DevOps
+
+trigger: none
+pr: none
+
+stages:
+- stage: Run
+
+  variables:
+  - template: ./templates/variables/globals.yml
+
+  jobs:
+  - job: Run
+    strategy:
+      matrix:
+        NET:
+          PathPrefix: 'net'
+        Python:
+          PathPrefix: 'python'
+        JS:
+          PathPrefix: 'js'
+        Java:
+          PathPrefix: 'java'
+
+    pool:
+      vmImage: ubuntu-16.04
+
+    variables:
+      Organization: azure-sdk
+      Project: internal
+
+    steps:
+
+      - task: DotNetCoreInstaller@0
+        displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+        inputs:
+          version: '$(DotNetCoreVersion)'
+
+      # TODO: Publish package to a private feed so we don't have to build from source
```

Comment:
Do you have an issue tracking this TODO?

Reply:
https://github.com/azure/azure-sdk-tools/issues/119